### PR TITLE
Master-data database infrastructure

### PR DESCRIPTION
**Pull request by kristianschneider**
This creates the database for master data storage

### CODE REVIEW

File: build/primary/main/locals.tf [hunk 1]
```diff
@@ -26,4 +26,5 @@ locals {
   MASTER_DATA_PATH_ES_BRP_RELATIONS             = "es-brp-relations"
   MASTER_DATA_PATH_GRID_LOSS_SYSTEM_CORRECTION  = "grid-loss-system-correction"
   MS_DATABASE_CONNECTION_STRING                 = "Server=tcp:${data.azurerm_key_vault_secret.mssql_data_url.value},1433;Initial Catalog=${module.mssqldb_aggregations.name};Persist Security Info=False;User ID=${data.azurerm_key_vault_secret.mssql_data_admin_name.value};Password=${data.azurerm_key_vault_secret.mssql_data_admin_password.value};MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;"
+  MS_DATABASE_MASTERDATA_CONNECTION_STRING      = "Server=tcp:${data.azurerm_key_vault_secret.mssql_data_url.value},1433;Initial Catalog=${module.mssqldb_aggregations_masterdata.name};Persist Security Info=False;User ID=${data.azurerm_key_vault_secret.mssql_data_admin_name.value};Password=${data.azurerm_key_vault_secret.mssql_data_admin_password.value};MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;"
```
Comment (BjarkeMeier) [inline build/primary/main/locals.tf:29]: Just curious: What is the "MS_" underscore prefix for?

File: build/primary/main/mssqldb-this.tf [hunk 2]
```diff
@@ -25,5 +25,17 @@ module "mssqldb_aggregations" {
   environment_instance  = var.environment_instance
   server_id             = data.azurerm_mssql_server.mssqlsrv.id
   
+  tags                  = azurerm_resource_group.this.tags
+}
+
+module "mssqldb_aggregations_masterdata" {
```
Comment (BjarkeMeier) [inline build/primary/main/mssqldb-this.tf:31]: Does it add any value to "aggregations" to all names? If not I'd prefer the shorter "mssql_masterdata".

File: build/primary/main/outputs.tf [hunk 3]
```diff
@@ -24,6 +24,12 @@ output ms_aggregations_connection_string {
   sensitive   = true
 }
 
+output ms_aggregations_masterdata_connection_string {
```
Comment (BjarkeMeier) [inline build/primary/main/outputs.tf:27]: How about?
```suggestion
output masterdata_connection_string {
```

File: build/primary/main/locals.tf (hunk 1 above)
Comment (kristianschneider) [reply build/primary/main/locals.tf:29]: Microsoft I think

File: build/primary/main/mssqldb-this.tf (hunk 2 above)
Comment (kristianschneider) [reply build/primary/main/mssqldb-this.tf:31]: In my opinion it makes it more descriptive

**Review by ZavezX — APPROVED** [review]

**Pull request merged**